chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -117,11 +131,27 @@ public ActivityDetails activityById(@PathVariable("id") final Long id)
     }
 
     @ApiOperation(value = "Get metadata about auditable records in the system for listings.",
+            notes = "All parameters are optional and will default to the first page of listing activity"
+                    + "from the beginning of CHPL with a page size of the maximum allowed. Page number is 0-based.")
```

Comment:
Does this default to "from the beginning" or "from the end" of CHPL?

Reply:
From "now".. I forgot we had changed the sort.. pushed a fix.